flink-examples/flink-examples-streaming/pom.xml

```diff
@@ -360,6 +360,26 @@ under the License.
 						</configuration>
 					</execution>
 
+					<!--Kafka09-->
+					<execution>
+						<id>Kafka09</id>
+						<phase>package</phase>
+						<goals>
+							<goal>jar</goal>
+						</goals>
+						<configuration>
+							<classifier>Kafka09</classifier>
+							<archive>
+								<manifestEntries>
+									<program-class>org.apache.flink.streaming.examples.kafka.ReadFromKafka09</program-class>
+								</manifestEntries>
+							</archive>
+							<includes>
+								<include>org/apache/flink/streaming/examples/kafka/ReadFromKafka09.class</include>
+								<include>org/apache/flink/streaming/examples/kafka/WriteToKafka09.class</include>
+							</includes>
+						</configuration>
+					</execution>
```

Comment:
Yes. My bad. I did not check what examples can be executed via command line. And I shouldn't have had to remove fat-jar packager from pom.xml file (see next comment). But this PR has many conflicts, so I close It and will create new one.